src/libs/agent-runtime/anthropic/index.ts

```diff
@@ -26,40 +24,22 @@ export class LobeAnthropicAI implements LobeRuntimeAI {
     this.client = new Anthropic({ apiKey });
   }
 
-  private buildAnthropicMessages = (
-    messages: OpenAIChatMessage[],
-  ): Anthropic.Messages.MessageParam[] =>
-    messages.map((message) => this.convertToAnthropicMessage(message));
-
-  private convertToAnthropicMessage = (
-    message: OpenAIChatMessage,
-  ): Anthropic.Messages.MessageParam => {
-    const content = message.content as string | UserMessageContentPart[];
-
-    return {
-      content:
-        typeof content === 'string' ? content : content.map((c) => this.convertToAnthropicBlock(c)),
-      role: message.role === 'function' || message.role === 'system' ? 'assistant' : message.role,
-    };
-  };
-
   async chat(payload: ChatStreamPayload, options?: ChatCompetitionOptions) {
     const { messages, model, max_tokens, temperature, top_p } = payload;
     const system_message = messages.find((m) => m.role === 'system');
     const user_messages = messages.filter((m) => m.role !== 'system');
 
-    const requestParams: Anthropic.MessageCreateParams = {
```

Comment:
is there any reason to change this part?

Reply:
@arvinxx just a small refactor to make the code more concise